ci-operator/step-registry/openshift/e2e/openstack/kuryr/openshift-e2e-openstack-kuryr-workflow.yaml

```diff
@@ -0,0 +1,20 @@
+workflow:
+  as: openshift-e2e-openstack-kuryr
+  steps:
+    pre:
+    - chain: ipi-openstack-pre
+    test:
+      - ref: openshift-e2e-test
+    post:
+    - chain: ipi-openstack-post
+    env:
```

Comment:
If we decide to reuse the osp cloud probably we should specify the OS_CLOUD(kuryr-cloud)

Reply:
We have in our secrets vault three separate “clouds.yaml” files, each one of which has only one cloud with name “openstack”.
 In the ipi-conf-openstack-ref ref, we mount them all and we pick the one that is mounted in the directory that has the name of the cluster-profile.

As a result, all following steps have a clouds.yaml file in the same well-known shared directory, with the same cloud name, but the contents will differ based on the cluster-type we pass in “config”

Reply:
I see, cool